src/devices/Ads1115/ComparatorQueue.cs

```diff
@@ -5,13 +5,17 @@
 namespace Iot.Device.Ads1115
 {
     /// <summary>
-    /// Set Comparator Queue
+    /// Comparator Queue
```

Comment:
Here and elsewhere: All sentences should end with a dot.

Reply:
Actually some of the stuff I added xml docs to I wasn't sure how it was supposed to be used but if I wanted to document everything super accurately I'd have to spend week-month instead of a day. Since the main goal was to enforce xml docs I'd prefer we do first pass to enable global flag and then incrementally improve it from there (so that new PRs will not cause build breaks etc)